src/components/Events/PublishedScheduleTable.jsx

```diff
@@ -2,26 +2,36 @@ import { NPOBackend } from '../../utils/auth_utils.js';
 import Events from './Events.jsx';
 import EventInfo from './EventInfo.jsx';
 import PropTypes from 'prop-types';
-
 import { useEffect, useState } from 'react';
-
-import { Table, Thead, Tbody, Tr, Th, Td, TableContainer, Box } from '@chakra-ui/react';
+import { Table, Thead, Tbody, Tr, Th, Td, TableContainer, Box, IconButton, useDisclosure } from '@chakra-ui/react';
+import AddDayModal from '../../pages/PublishedSchedule/AddDayModal.jsx'
+import { AddIcon } from '@chakra-ui/icons';
 
 const PublishedScheduleTable = ({ season }) => {
   const [eventsInDay, setEventsInDay] = useState([]);
   const seasonType = season.split(' ')[0];
   const seasonYear = season.split(' ')[1];
+  const [dataShouldRevalidate, setShouldDataRevalidate] = useState(false);
+  const { isOpen: isOpenDay, onOpen: onOpenDay, onClose: onCloseDay } = useDisclosure();
+
+  const renderTable = async () => {
+    const { data } = await NPOBackend.get(
+      `/published-schedule/season?season=${seasonType}&year=${seasonYear}`,
+    );
+    setEventsInDay(data);
+  };
 
   useEffect(() => {
-    const renderTable = async () => {
-      const { data } = await NPOBackend.get(
-        `/published-schedule/season?season=${seasonType}&year=${seasonYear}`,
-      );
-      setEventsInDay(data);
-    };
     renderTable();
   }, [seasonType, seasonYear]);
 
+  useEffect(() => {
+    if (dataShouldRevalidate) {
+      renderTable();
+      setShouldDataRevalidate(false);
+    }
+  }, [dataShouldRevalidate])
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'renderTable'. Either include it or remove the dependency array.